Updated Swedish translation

mcatanzaro opened the pull request:
#### 8144b31187bf4a5b8a204f32b1a5d0c9700a9aab
<pre>
Updated Swedish translation
<a href="https://bugs.webkit.org/show_bug.cgi?id=241759">https://bugs.webkit.org/show_bug.cgi?id=241759</a>

Rubber-stamped by Michael Catanzaro.

* Source/WebCore/platform/gtk/po/sv.po:

Canonical link: <a href="https://commits.webkit.org/251909@main">https://commits.webkit.org/251909@main</a>
</pre>

webkit-commit-queue commented:
'Michael Catanzaro <mcatanzaro@redhat.com>' cannot review their own change, blocking PR #1654

mcatanzaro commented:
Too smart....

mcatanzaro commented:
Oops... wasn't waiting for review here, I just forgot to reset the labels.

haaninjo committed:
Updated Swedish translation
https://bugs.webkit.org/show_bug.cgi?id=241759

Rubber-stamped by Michael Catanzaro.

* Source/WebCore/platform/gtk/po/sv.po:

Canonical link: https://commits.webkit.org/251909@main
